packages/manager/modules/pci/src/projects/project/storages/databases/database/pools/information/translations/Messages_fr_FR.json

```
@@ -0,0 +1,11 @@
+{
+  "pci_databases_pools_information_title": "Informations du Pool",
+  "pci_databases_pools_information_database": "Base de donnée",
```

See: packages/manager/modules/pci/src/projects/project/storages/databases/database/pools/add-edit/translations/Messages_fr_FR.json (pci_databases_pools_add_database), it's written with an `s`.

```suggestion
  "pci_databases_pools_information_database": "Base de données",
```